.github/workflows/INTEGRATION_TEST.yml

```diff
@@ -59,23 +59,23 @@ jobs:
           fi
 
       - name: Determine Helm chart ref to use
-        id: helm-branch
+        id: helm-dir
         run: |
-          if [ -z "${{ inputs.helm-branch }}" ]; then
-            helm_branch=$(jq -r ".[\"${{ inputs.release-branch || github.ref_name }}\"]" .github/workflows/helm-git-refs.json)
-            if [ -z "$helm_branch" ] || [ "$helm_branch" == "null" ]; then
-              echo "::error::Could not determine Helm chart branch to use, please provide helm-branch input or adjust the mappings in .github/workflows/helm-git-refs.json"
+          if [ -z "${{ inputs.helm-dir }}" ]; then
+            helm_dir=$(jq -r ".[\"${{ inputs.release-branch || github.ref_name }}\"]" .github/workflows/helm-git-refs.json)
+            if [ -z "$helm_dir" ] || [ "$helm_dir" == "null" ]; then
+              echo "::error::Could not determine Helm chart dir to use, please provide helm-dir input or adjust the mappings in .github/workflows/helm-git-refs.json"
               exit 1
             fi
-            echo "helm-branch=$helm_branch" >> $GITHUB_OUTPUT
+            echo "helm-dir=$helm_dir" >> $GITHUB_OUTPUT
           else
-            echo "helm-branch=${{ inputs.helm-branch }}" >> $GITHUB_OUTPUT
+            echo "helm-dir=${{ inputs.helm-dir }}" >> $GITHUB_OUTPUT
           fi
```

Comment:
Good catch!

Happy to make at least these changes 😄 

I'll implement (and also re-check my contribution for readability) after lunch. If you think of any other improvements before then, please let me know!

Reply:
@chillleader I addressed your two suggestions and I also changed the `connectors-version` step ID to `determine-connectors-version`.

I personally think having the workflow input and the step output be the same is more readable. Especially since the step output is always fully qualified as `*.outputs.helm-dir`

What do you think? I'm open to making additional edits here 😄

Reply:
Works for me!